__tests__/Unit/pages/Progress/ProgressUpdatesPage.test.tsx

```diff
@@ -16,7 +16,10 @@ describe('Progress page test', function () {
         );
         const textBoxes = screen.getAllByRole('textbox');
         expect(textBoxes).toHaveLength(3);
-        const button = screen.getByRole('button') as HTMLButtonElement;
+        const button = getByRole(
+            screen.getByRole('form') as HTMLFormElement,
+            'button'
+        ) as HTMLButtonElement;
```

Comment:
why was this added? this is a part of progress updates, why should it be concerned with the navbar?

Reply:
So now you check only the button inside the form i.e. submit button...before it check all the button in progressUpdatePage?? Is I am right??

Reply:
No, since the test is written targetting the progressUpdatePage's form element (because when the test is written there is no login button that render in progressUpdatePage) (AFAIK) checking that there is one button in one form & i dont want to break the test by ignoring it.
So i rewritten the test in the same way it used to work, so now it ignores NavBar & NavBar has seperate test https://github.com/Real-Dev-Squad/website-status/blob/develop/__tests__/Unit/Components/Navbar/Navbar.test.tsx

**So now this test only checks that there will be only one button in one form & there is only one form in one progressUpdatePage component**

Reply:
This is not a part of this Issue, please handle it separately